modules/apps/27-interchain-accounts/types/keys.go

```diff
@@ -6,22 +6,34 @@ const (
 	// ModuleName defines the Interchain Account module name
 	ModuleName = "interchainaccounts"
 
-	// Version defines the current version the IBC tranfer
+	// Version defines the current version the IBC interchainaccounts
 	// module supports
 	Version = "ics27-1"
 
+	// PortID is the default port id that the interchainaccounts module binds to
 	PortID = "ibcaccount"
 
-	StoreKey  = ModuleName
+	// StoreKey is the store key string for IBC interchainaccounts
+	StoreKey = ModuleName
+
+	// RouterKey is the message route for IBC interchainaccounts
 	RouterKey = ModuleName
 
+	// QuerierRoute is the querier route for IBC interchainaccounts
+	QuerierRoute = ModuleName
+)
+
+var (
+	// ModuleAccountKey defines the key used to construct to the IBC interchainaccounts module account address
+	ModuleAccountKey = []byte(ModuleName)
+
 	// Key to store portID in our store
 	PortKey = "portID"
```

Comment:
Should this align with the `transfer` module?

```
PortKey = []byte{0x01}
```

[See here](https://github.com/cosmos/ibc-go/blob/main/modules/apps/transfer/types/keys.go#L36)

Reply:
I was wondering this also tbh

Reply:
I think we should opt for consistency either way! I'll update this to align with `transfer` for now. We can revisit and discuss during the internal audit